Automattic/wp-calypso
# Show "Get Business" CTA for trial sites plan upgrade

Closes https://github.com/Automattic/dotcom-forge/issues/4450

## Proposed Changes

* Check if current plan is a business trial plan and show "Get Business" on the upgrade page

## Testing Instructions



- On a trail site go to `/plans/siteSlug`
- Verify business plan CTA is` Get Business` and not `Upgrade`

![image](https://github.com/Automattic/wp-calypso/assets/47489215/8bc48347-152e-445d-a599-db91ae8158be)

## Pre-merge Checklist
